### ISSUE TITLE
Screen Reader: The captions button should announce it’s state

### ISSUE THREAD

**Issue by miesgre** [polimediaupv/paella-core#297]
The captions button should announce it’s state. It’s not saying if closed captions are on or off.

@karendolan
